backend/backend/urls.py

```diff
@@ -1,22 +1,19 @@
-"""API URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/2.1/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
+from django.urls import include, path
+from django.conf.urls import url, include
+from rest_framework import routers
+from authentication import views
 from django.contrib import admin
-from django.urls import path, include
 
+router = routers.DefaultRouter()
+router.register(r'users', views.UserViewSet)
+router.register(r'groups', views.GroupViewSet)
+router.register(r'userprofile', views.UserProfileViewSet)
```

Comment:
Let's delete this and fold this into the user routes.

Reply:
WIP

Reply:
We finished it!